HIP/hip-920.md

```diff
@@ -0,0 +1,252 @@
+          "properties": {
+            "streaming_royalties": 5,
+            "streaming_royalty_token": "0.0.988765",
+            "token_gated": true,
+            "track_number": 1,
+            "bpm": 120,
+            "duration": 240,
+            "key": "A Minor",
+            "location_created": "London, UK",
+            "recording_studio": "Digital Sound Studios",
+            "license": "CC BY-NC-ND 4.0",
+            "copyright": "MetaHeads, 2024",
+            "label": "Digital Rock Records",
+            "master": ["Digital Mastering Services"],
+            "publisher": ["MetaMusic Publishing"],
+            "pro": ["ASCAP"],
+            "release_date": "2024-01-01T00:00:00Z",
+            "isrc": "QMJMT2047001",
+            "lyrics": "[00:00:00] Echoes in the circuitry... [00:00:30] A digital awakening...",
+            "album": "Echoes of the Digital Age",
+            "upc": "123456789012",
+            "artist": "MetaHeads",
+            "genre": {
+              "root": "Rock",
+              "sub": ["Electronic"]
+            },
+            "credits": {
+              "producer": "Max Soundwave",
+              "mix": "Eddie Echo",
+              "composer": "MetaHeads",
+              "performers": ["Max Soundwave", "Eddie Echo", "Tina Tempo", "Bassline Ben"],
+              "songwriter": "MetaHeads"
+            }
+          }
+        }
+      },
+      {
+        "uri": "ipfs://track2-uri",
+        "type": "audio/mpeg",
+        "metadata": {
+          "name": "Circuit Breaker",
+          "description": "A powerful track about breaking free from digital constraints.",
+          "properties": {
+            // Similar metadata as for Track 1, with relevant changes for track number, ISRC, etc.
+          }
+        }
+      },
+      {
+        "uri": "ipfs://track3-uri",
+        "type": "audio/mpeg",
+        "metadata": {
+          "name": "Virtual Horizon",
+          "description": "The closing track reflecting on the digital journey.",
+          "properties": {
+            // Similar metadata as for Track 1, with relevant changes for track number, ISRC, etc.
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+
+## Backwards Compatibility
+
+This proposal aims to extend HIP-412 without disrupting existing standards. It ensures backward compatibility with current NFT metadata structures.
+
+## Security Implications
+
+The standard does not directly impact security but emphasizes the importance of accurate and immutable metadata for NFTs, which is crucial for rights and royalty management in music assets.
+
+## How to Teach This
+
+Educational materials will be developed to assist artists, record labels, and NFT creators in understanding and implementing the new Music Metadata Standard in their digital assets.
+
+## Reference Implementation
+
+Multiple projects and artists have already incorporated this working music metadata standard.
```

Comment:
Since this is largely based on how the structure is defined/ validated for the metadata, I think that there is low hanging fruit here in terms of a reference implementation - specifically I'm thinking of adding a schema. E.g. [JSON schema](https://json-schema.org/)

Reply:
i don't know if this is what you mean, but you can see these working standards live on an NFT project i did as an example: https://dd.style/hellofuture

Reply:
Yes, I think including a link to this project (and others implementing this standard) would be a great idea.
in addition to the website, perhaps also link to it on hashscan (e.g. https://hashscan.io/mainnet/token/0.0.2997600 )
> you can see these working standards live on an NFT project i did as an example: https://dd.style/hellofuture

My suggestion here was something different though - it was to include a schema validator as part of the HIP.  Essentially, take the tables describing each of the fields of the metadata, and turn that into a schema. So that would mean that you can take the schema plus the metadata, and pass those as inputs into a validator, and it would output whether it conforms.
> i don't know if this is what you mean,

Reply:
VERY valuable feedback thanks so much. im going to take your 3 pieces of feedback so far and make sure the working group is aware (we have a call tomorrow). i am a big fan, but this feels the most urgent to update the HIP with proper examples.

i was also realizing i need to fill out the "requested by" field because there are a few more projects/individuals to add.

Reply:
just updating you here @bguiz the group greatly appreciated this feedback, a top priority now.

i have been slammed, but will get the HIP updated with this stuff as soon as i can : )